okio/src/test/kotlin/okio/ByteStringTest.kt

```diff
@@ -111,10 +157,246 @@ class ByteStringTest {
   }
 
   @Test fun indexOfByteArray() {
-    val byteString = "112233".decodeHex()
+    val byteString = factory.decodeHex("112233")
     assertEquals(0, byteString.indexOf("112233".decodeHex().toByteArray()).toLong())
     assertEquals(1, byteString.indexOf("2233".decodeHex().toByteArray()).toLong())
     assertEquals(2, byteString.indexOf("33".decodeHex().toByteArray()).toLong())
     assertEquals(-1, byteString.indexOf("112244".decodeHex().toByteArray()).toLong())
   }
+
+  // TODO Kotlin/Native and Kotlin/JS have a null ByteString.EMPTY
```

Comment:
I think I’d prefer an `@Ignore` test instead of commented out code?

Reply:
... does kotlin.test have that?

Reply:
Yes

Reply:
I'm hesitant to disable the whole test because there are valuable assertions which should be tested. I can make a specific comment for each assertion I commented out with a reference to the Kotlin ticket I opened. Would that be acceptable?

Reply:
Or I can pull the 2 assertions out into a separate test and `@Ignore` it with a comment about the Kotlin ticket. That's probably what you wanted in the first place. I got there eventually.

Reply:
Yep!